common/lockmanager_test.go

```diff
@@ -0,0 +1,545 @@
+package common
+
+// Tests in this file will deal with locking, unlocking, and checking for stale locks (sometimes on multiple threads).
+// Since this can quickly get confusing, we will have comments for each test using the following syntax:
+// "%" denotes a thread off the main thread.
+// "!" denotes that the associated operation will produce an error.
```

Comment:
s/will/should/g

A small thing, but having "should" here indicates that we will always test that.